Codeinwp/neve-fse
# Fix for Main group

### Summary

This fixes the issue where the main group for all the templates, has some padding on the sides in the editor.
After the fix, there should be no padding in the Editor.

### Will affect visual aspect of the product

YES, In the Editor only

___
### Screenshots 

![Screenshot 2023-07-13 at 09 51 33](https://github.com/Codeinwp/neve-fse/assets/52494172/27d28b60-c614-4150-9bba-fee8f5888324)

___

![Screenshot 2023-07-13 at 09 53 04](https://github.com/Codeinwp/neve-fse/assets/52494172/c61dc928-b604-4cfb-b5f7-9bc9ae7e4a34)


___
### Test instructions

- Install the build
- open a few templates like 404, front page, index, etc
- see if the main group has **Enabled Inner blocks use content width** and/or a transparent bg color set
- in case the 2 things mentioned above are checked -> the Main Group shouldn't have any padding on the side in the Editor


Closes #54 .
